modules/microshift-migrating-ipv6-dual-stack-config.adoc

```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-migrating-ipv6-dual-stack-config_{context}"]
+= Migrating a {microshift-short} to IPv6 dual-stack networking
+
+You can convert your IPv4 single-stack cluster to dual-stack cluster networking that supports IPv4 and IPv6 address families by setting two entries in the service and cluster network parameters in the {microshift-short} configuration file.
```

Comment:
You can also convert from IPv6 to IPv6+IPv4, but not sure if this is something any customer would do.